ISM43362/ISM43362.cpp

```diff
@@ -111,12 +111,23 @@ bool ISM43362::reset(void)
     _resetpin = 1;
     wait_ms(500);
 
-    /*  Wait for prompt line */
-    if (!_parser.recv("> \r\n")) {
```

Comment:
Can't we use as well the method used in next commit ? 
Like using ">%[^\n]\n"  ? that could make it simpler

Reply:
Ok to use ">[%\r\n]", without \r\n at the end.
Kind regards